WISE-Community/WISE
# Added the Features page

Added content to /features and also updated a bunch of site styles. Added a new font-family for some headings (Raleway).

To test:
- Load the site and navigate to `/features`.
- Review the content and make sure their are no style or layout bugs with all different browser sizes.
- Navigate around the rest of the site and look for layout and style bugs or any other issues.

Closes #1464.
